praw/models/reddit/subreddit.py

```diff
@@ -17,6 +17,117 @@ class Subreddit(RedditBase, MessageableMixin, SubredditListingMixin):
     STR_FIELD = 'display_name'
     MESSAGE_PREFIX = '#'
 
+    @staticmethod
+    def _create_or_update(_reddit, allow_images=None, allow_top=None,
+                          collapse_deleted_comments=None,
+                          comment_score_hide_mins=None, description=None,
+                          domain=None, exclude_modqueue_banned=None,
+                          header_hover_text=None, hide_ads=None, lang=None,
+                          key_color=None, link_type=None, name=None,
+                          over_18=None, public_description=None,
+                          public_traffic=None, show_media=None,
+                          spam_comments=None, spam_links=None,
+                          spam_selfposts=None, sr=None, submit_link_label=None,
+                          submit_text=None, submit_text_label=None,
+                          subreddit_type=None, suggested_comment_sort=None,
+                          title=None, wiki_edit_age=None, wiki_edit_karma=None,
+                          wikimode=None):
+        """Create a new subreddit.
+
+        :param _reddit: The reddit session.
+
+        :param allow_images: Allow users to upload images using the native
+            image hosting. Only applies to link-only subreddits.
+        :param allow_top: Allow the subreddit to appear on /r/all as well as
+            the default and trending lists.
+        :param collapse_deleted_comments: Collapse deleted and removed comments
+            on comments pages by default.
+        :param comment_score_hide_mins: The number of minutes to hide comment
+            scores.
+        :param description: Shown in the sidebar of your subreddit.
+        :param domain: Undocumented.
+        :param exclude_modqueue_banned: Exclude posts by site-wide banned users
+            from modqueue/unmoderated.
+        :param header_hover_text: The text seen when hovering over the snoo.
+        :param hide_ads: Don't show ads within this subreddit. Only applies to
+            gold-user only subreddits.
+        :param key_color: A 6-digit rgb hex color (e.g. `#AABBCC`), used as a
+            thematic color for your subreddit on mobile.
+        :param lang: A valid IETF language tag (underscore separated).
+        :param link_type: The types of submissions users can make.
+            One of ``any``, ``link``, ``self``.
+        :param name: The name for the new subreddit. Required during creation.
+        :param over_18: Viewers must be over 18 years old.
```

Comment:
Can we put, "i.e., NSFW" since that's what people commonly refer to this setting as?